src/ui/components/Library/Team/View/NewTestRuns/TestRunSpecDetails.tsx

```diff
@@ -116,12 +117,26 @@ function ErrorGroup({
       className="flex w-full flex-col gap-2 overflow-x-auto rounded-md bg-[color:var(--testsuites-v2-error-bg)] px-3 py-4"
       data-test-id="TestRunSpecDetails-Error"
     >
-      <button className="flex flex-row gap-1" onClick={() => setExpanded(!expanded)}>
-        <div>({count})</div>
-        <div className="truncate">{summary}</div>
+      <button
+        className="flex flex-row items-start justify-between gap-1"
+        onClick={() => setExpanded(!expanded)}
+      >
+        <div className="mt-0.5 flex items-center justify-center rounded-md bg-[color:var(--testsuites-failed-color)] px-2 py-0 text-xs font-bold text-white">
+          {count}
+        </div>
+        <div className="ml-2 flex-grow truncate text-left">{summary}</div>
+        <Icon
+          data-test-id="TestRunResults-StatusGroup-Icon"
+          data-test-state={expanded ? "expanded" : "collapsed"}
+          className={`duration-140 mt-0.5 transition-transform ${
+            expanded ? "rotate-0" : "rotate-90"
+          }`}
+          style={{ minWidth: "1rem", minHeight: "1rem" }}
```

Comment:
I think we need to use explicit `height` here. The horizontal flex box doesn't handle `min-width` well with scalable content like an SVG. Also, if we need to use inline styles, that seems like a good trigger to switch to using CSS modules rather than mixing tailwind and inline styles.

<img width="601" alt="image" src="https://github.com/replayio/devtools/assets/788456/19c469b9-f929-4cd1-893e-b4191d537278">

Reply:
Whoops, fixed.

CSS: I propose pushing this out with the inline stuff as-is, then I'll address in a follow-on PR.